jax_rocm_plugin/build/rocm/tools/build_wheels.py

```diff
@@ -159,6 +159,9 @@ def build_jaxlib_wheel(
         "python",
         "build/build.py",
         "build",
+        "--bazel_options=--strip=never",
```

Comment:
I'd prefer that as many Bazel options as possible make their way down to the `jax_rocm_plugin/.bazelrc` file rather than getting threaded through one of our dozen-or-so wrapper scripts. Preferably, it'd be inside of something like a `build:debug` config, so you can enable it with `./bazel build ... --config=debug ...` or `build/buid.py ... --bazel_options="--config=debug" ...`.